compliance_tool/cli.py

```diff
@@ -1,57 +1,67 @@
 import argparse
-from compliance_tool import list, scan, remediate
+from compliance_tool import list_, scan, remediate
 
 
-def prepare_parser_list(subparsers) -> None:
-    parser = subparsers.add_parser("list",
-                                   help="list available components")
-    parser.set_defaults(func=list.execute)
+def prepare_parser_list(cli_sub_parsers, global_parser, profile_parser) -> None:
+    parser = cli_sub_parsers.add_parser("list", parents=[global_parser],
+                                        help="list available components")
```

Comment:
It should list available security profiles and their identifiers.

Moreover, nobody knows what a "component" means.

Reply:
I've expanded the message a bit.